Backendless/WebORB/Client/HttpEngineWithClient.cs

```diff
@@ -23,14 +23,26 @@ public HttpEngineWithClient( String url, IdInfo idInfo ) : base( url, idInfo )
 
     public override void SendRequest<T>( V3Message v3Msg, IDictionary requestHeaders,
                                          IDictionary httpHeaders, Responder<T> responder,
-                                         AsyncStreamSetInfo<T> asyncStreamSetInfo ) =>
-        SendHttpRequest( v3Msg, requestHeaders, httpHeaders, responder, asyncStreamSetInfo );
+                                         AsyncStreamSetInfo<T> asyncStreamSetInfo )
+    {
+      try
+      {
+        Task task = SendHttpRequest( v3Msg, requestHeaders, httpHeaders, responder, asyncStreamSetInfo );
+        task.Wait();
+      }
+      catch( System.Exception ex )
+      {
+        throw ex;
```

Comment:
@holailusoria  this makes no sense. To catch an exception only to rethrow it is an example of a code that does absolutely nothing.

Reply:
@markpiller , I removed the extra code.